script/release-publish.sh

```diff
@@ -60,4 +66,5 @@ hub release create \
     --attach="$ZIPF" --attach="$ZIPF_NO_VER" \
     --attach="$TGZF" --attach="$TGZF_NO_VER" \
     --message="$TAG_NAME" --message="$body" \
+    --attach="$SHA256F" --message="$(cat \"$SHA256F\")" \
```

Comment:
attachments and messages together:

```suggestion
    --attach="$TGZF" --attach="$TGZF_NO_VER" --attach="$SHA256F" \
    --message="$TAG_NAME" --message="$body" --message="$(cat \"$SHA256F\")" \
```

Incidentally, why do we need to he hash both as an attachment and as a message in the release notes? (just curious)

Reply:
1. Attachments and messages together look good. :+1:
2. I have seen some releases in the wild to be like this.

Example: https://github.com/cosmos/gaia/releases/latest
They include `build_report` in the release body.

Probably people prefer copying into clipboard from the browser rather than from a separate downloaded file.

Reply:
I also wanted to ask your opinion if I should change `--message="$(cat \"$SHA256F\")"` to `checksums=$(cat "$SHA256F")` and `--message="$checksums"`.

Reply:
I'm OK with the inline cat here. But would be ok with factoring that into a temp variable too if you preferred.

I'm not sure of what the benefit is here tho. You have the checksum available either way, but including it in the release message seems to clutter human-readible communications with machine readable info. I'd prefer we just upload the file and not clutter or release notes.

Reply:
> but including it in the release message seems to clutter human-readible communications with machine readable info

Agreed, I also can't think of a compelling reason to have it inline.
I'd leave it out for now, we can always paste it on if we discover a use case.

Reply:
We can also parse the `sha256sum.txt` and put it as markdown tables. Example, [tlaplus release](https://github.com/tlaplus/tlaplus/releases/tag/v1.7.2).

Nonetheless, it is not that important to me. I will be totally ok if you decide to keep `sha256sum.txt`` as a release artifact.